python/graphstorm/model/gnn.py

```diff
@@ -395,6 +412,22 @@ def set_loss_func(self, loss_fn):
                 'The loss function should be the class of nn.Module.'
         self._loss_fn = loss_fn
 
+    def set_loss_func_dict(self, loss_fns):
```

Comment:
I just add this function here, but I did this function when creating graphstorm's model.

Reply:
the same comment as above

Reply:
this function isn't used to set different loss functions for different node types.

Reply:
solved. In the update, variables `config.num_classess`, `config.multilabel`, `config.multilabel_weights`, and `config.imbalance_class_weights` can be defined as a per node type dictionary.

However, the current implementation only support node classification but does not support node regression, because those variables are not used in regression task in graphstorm.